src/components/ImageGrid/ImageGridComponent.tsx

```diff
@@ -98,26 +105,48 @@ const ImageGridComponent = ({
     } else {
       setRowCol({ col: 1, row: 1 });
     }
-  }, [numberOfImgs, images]);
+  }, [numberOfImgs, randomPhotos]);
+
+  type unsplashPhotoFortmat = {
```

Comment:
Prefer to use `interface` instead of type for typing objects

Reply:
Also please note, in TS, types, interfaces and enums names are capitalized, so `UnsplashPhotoFormat`